src/foundry/client/canvas/animation/canvas-animation.d.mts

```diff
@@ -44,10 +44,12 @@ declare class CanvasAnimation {
    * ];
    * CanvasAnimation.animate(attributes, {duration:500});
    * ```
+   *
+   * @privateRemarks `extends object` to allow {@linkcode foundry.canvas.interaction.Ping._animateFrame | Ping#_animateFrame} to take `CanvasAnimation.AnimationData<this>`
    */
-  static animate<AnimationParent extends AnyObject = AnyObject>(
-    attributes: CanvasAnimation.Attribute<AnimationParent>[],
-    options?: CanvasAnimation.AnimateOptions<AnimationParent>,
+  static animate<Parents extends object[] = AnyObject[]>(
```

Comment:
I'd prefer `object[] = object[]`. The weakest trait of `AnyObject` is the ability to access any property on it as `unknown` since it can mask typos. If `AnyObject` isn't the bound it basically has no benefits.